src/AutoBalance.cpp

```diff
@@ -108,10 +108,10 @@ class AutoBalanceMapInfo : public DataMap::Base
 static std::map<int, int> forcedCreatureIds;
 // cheaphack for difficulty server-wide.
 // Another value TODO in player class for the party leader's value to determine dungeon difficulty.
-static int8 PlayerCountDifficultyOffset, LevelScaling, higherOffset, lowerOffset, numPlayerConf;
```

Comment:
numPlayerConf isn't used? (maybe it's from that recent commit from Yehonal where he removed the config option, dunno)

Reply:
Yehonal removed it from the config file because it was never used in the first place (see commit e5c6345). But he forgot to also remove it from the code.